xdsl/dialects/builtin.py

```diff
@@ -1244,7 +1244,7 @@ def __init__(self, ops: List[Operation] | Region):
 
     @property
     def ops(self) -> List[Operation]:
```

Comment:
Actually yes, this should return the iterator instead.

Reply:
Upcoming PR :)

Reply:
Okay then fine!